### PR TITLE
Added Cancel Buttons to Create/Edit pages

### PR DESCRIPTION
**Pull request by KavsHub**
As requested, there's now a cancel button on the create/edit pages.

### CODE REVIEW

**Review by JackSekoranja — CHANGES_REQUESTED** [review]
Works well, but could you do the same for program/subplan/course deletion? After that I believe that will be everything we need

**Comment by KavsHub** [comment]
> Works well, but could you do the same for program/subplan/course deletion? After that I believe that will be everything we need

I thought there was already yes - no buttons but apparently not anymore. I've updated them now

**Review by JackSekoranja — APPROVED** [review]
Looks good, happy for you to merge

File: cassdegrees/static/js/main.js [hunk 1]
```diff
@@ -10,4 +10,18 @@
...
+function goBack() {
+    window.location.href= "/";
+}
+
+/**
+ * Returns to the relevant list page
+ */
+function returnToList(pageName) {
+    window.location.href = "/list/?view="+pageName;
 }
```
Comment (j-selby) [inline cassdegrees/static/js/main.js]: Newline

**Pull request merged**